lua/precognition/init.lua

```diff
@@ -335,10 +220,7 @@ local function on_insert_enter(ev)
 end
 
 local function on_buf_edit()
-    apply_gutter_hints(
-        build_gutter_hints(vim.api.nvim_get_current_buf()),
-        vim.api.nvim_get_current_buf()
-    )
+    apply_gutter_hints(build_gutter_hints(), vim.api.nvim_get_current_buf())
```

Comment:
Same here, maybe `apply_gutter_hints` should just assume curbuf?

Reply:
agree